File: Repositories/CSI-Java-2021-01/CSI-Antonio-Negron/MainProject/src/csi/negron/pkg/RockPaperScissors.java
package csi.negron.pkg;


	import java.util.*;
	import java.util.Scanner;

	public class RockPaperScissors {
	  public static void main(String[] args) {  
		  Random r=new Random();
	   String[]choices= {"Rock","Paper","Scissor"};
	   String computerChoice=choices[r.nextInt(choices.length)];
	   Scanner sc=new Scanner(System.in);
	   
	
	   
	   System.out.println("Type your choice:");
	   int inputInt=Arrays.asList(choices).indexOf(sc.nextLine());
	   String userChoice=choices[inputInt];
	   
	   System.out.println("Computer chose"+ computerChoice);
	   System.out.println("User Chose"+userChoice);
	   
	   if(computerChoice==userChoice) {
		   System.out.println("Its a Tie!");
	   }
	   else if(computerChoice==choices[0]&& userChoice==choices[1]) {
		   System.out.println("You Win!");
	   }
	   else if(computerChoice==choices[1]&& userChoice==choices[0]) {
		   System.out.println("You Lost!");
	   }
	   else if(computerChoice==choices[0]&& userChoice==choices[2]) {
		   System.out.println("You Lost!");
	   }
	   else if(computerChoice==choices[2]&& userChoice==choices[0]) {
		   System.out.println("You Win!");
	   }
	   else if(computerChoice==choices[1]&& userChoice==choices[2]) {
		   System.out.println("You Win!");
	   }
	   else if(computerChoice==choices[2]&& userChoice==choices[1]) {
		   System.out.println("You Lost!");
	   }
	   else {
		   System.out.println("Something Went Wrong");
	   }
	  } 
	}
	

	

File: Repositories/CSI-Python-2021-02/CSI-Andres-Juncos/Modules/Module5/RockPaperScissors.py
import random

userchoice = input("What is your choice?")

foo = ['Rock', 'Paper', 'Scissors']
computerChoice = random.choice(foo)
print(f"Computer selected: {computerChoice}")


if(userchoice == computerChoice):
    print("Its a tie")
elif(userchoice=='Rock' and computerChoice=='Scissors'):
    print("User won!")
elif(userchoice=='Paper'and computerChoice=='Rock'):
    print("User won!")
elif(userchoice=='Scissors' and computerChoice=='Paper'):
    print("User won!")
elif(userchoice=='Paper' and computerChoice=='Scissors'):
    print("Computer won!")
elif(userchoice=='Scissors' and computerChoice=='Rock'):
    print("Computer won!")
elif(userchoice=='Rock' and computerChoice=='Paper'):
    print("Computer won!")


